packages/react-conformance-make-styles/README.md

```diff
@@ -0,0 +1,28 @@
+# @fluentui/react-conformance-make-styles
+
+A set of conformance tests for `@fluentui/react-make-styles`. Intended to be used with `@fluentui/react-conformance`.
+
+## Usage
+
+```ts
+import { isConformant } from '@fluentui/react-conformance';
+import makeStylesTests from '@fluentui/react-conformance-make-styles';
+
+isConformant({
+  Component,
+  extraTests: makeStylesTests,
```

Comment:
Hey the extensibility idea worked 😄 -- if you have any ideas for improvements to the API, I'm definitely open to that (it's still at v0 so we can break things)